source/guide_jingo.rst

```diff
@@ -0,0 +1,256 @@
+.. highlight:: console
+
+.. sidebar:: About
+
+  This guide has been tested with Uberspace 7.1.1 and Jingo 1.8.5
+
+#####
+Jingo
+#####
+
+Jingo_ is a simple Wiki software that is based on Node.js_ and Git. The content is stored in markdown files which are managed by a Git repository. In contrast to other Wiki software (like Mediawiki or Dokuwiki), Jingo does not provide too much functions and uses a very decent design. But because the management and versioning of the content is based on Git, it can be used in multiple ways.
+
+At this time, Jingo is no longer actively developed but still supported for security issues.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Node.js_ and its package manager npm_
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [isabell@stardust ~]$
+
+Your blog URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+First get the Jingo code from Github_:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/claudioc/jingo ~/jingo
```

Comment:
I am unsure if the git clone shut be done on master, or on the last release number like in the etherpad guide. Because here the releases are settled in tags, there would be necessary a bit more irritating git commands. 
The Jingo maintainer advises to pull from master anyway.

Reply:
If the Jingo maintainer advises to pull from master, that's fine.